docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -32,75 +42,113 @@ components:
 - **Layer 1 fee** - The L1 fee is the cost to publish your L2 transaction onto Ethereum and can 
```

Comment:
Let's not call it fees because it is not explicit to the user and can be misinterpreted. Rather, we can say that the gas price and gas used for inclusion on Linea covers the costs for the transaction to be included on the L1